src/manage.c

```diff
@@ -386,9 +388,11 @@ void manage_track_setup()
 {
   initialize_lock_counters();
 
-  SETTING_NOTIFY("track", "iq_output_mask", iq_output_mask, TYPE_INT,
+  SETTING_NOTIFY("expert", "iq_output_mask", iq_output_mask, TYPE_INT,
```

Comment:
There was talk of denoting the expert fields in the YAML. This would mean that you wouldn't have to group them all together.